src/components/ImageEdit/index.js

```diff
@@ -0,0 +1,111 @@
+import "!style!css!sass!./index.scss"
+
+import React from "react";
+import { injectIntl } from "react-intl"
+import Modal from "react-bootstrap/lib/Modal";
+import { RaisedButton } from "material-ui"
+import { postImage, deleteImage } from "src/actions/userImages.js"
+import { connect } from "react-redux"
+import FormFields from "../FormFields"
+import HelTextField from "../HelFormFields/HelTextField"
+
+class ImageEdit extends React.Component {
+
+    constructor(props) {
+        super(props)
+        this.state = {
+            name: this.props.defaultName || "",
+            photographerName: this.props.defaultPhotographerName || "",
```

Comment:
Here, when looking at an image that was POSTed successfully, the API reports the name of the photographer saved correctly, but the photographer name field is empty when I try to edit the image again.

Can you reproduce the bug?